### DISCUSSION TITLE
Set timeline_min_size=0 for streams

### DISCUSSION THREAD

**Discussion by FichteFoll** [tomasklaen/uosc#624] [Q&A] [answered]
I want to set `timeline_size_min=0`  whenever I open a stream. I can identify streams using the special profiles `[protocol.http]`, for example, and have tried something like the following (inspired from the comments in https://github.com/tomasklaen/uosc/issues/618):

```
[protocol.http]
script-opts-append=uosc-timeline_size_min=0
```

However, this doesn't do anything. I have verified that *other* mainline settings for mpv are applied when specified in this profile (or in a transitive one included via `profile=stream`, such as `interpolation=no`). I have also verified that the setting `script-opts-append=uosc-timeline_size_min=0` works when specified outside of a profile.

Is this an issue with uosc not reloading the setting or fetching it dynamically after mpv (& uosc) has initialized since the profile is only applied later once the file is being loaded? 

I've also not really found a way to set these options via a key binding. There is a special script message being registered for `set-min-visibility`, for example, but there isn't a generic one or one for this particular setting.

Edit: I suppose the `toggle-progress` message would do that for the binding part, but I'd still want it to not be visible by default for streams.

**Comment by christoph-heinrich** [comment]
I've updated #571 to make this work.

**Comment by FichteFoll** [comment]
On a somwhat related note, it appears that `toggle-progress` is not capable of forcefully showing the progress bar in fullscreen when `timeline_size_min_fullscreen=0` (the default). Is this something that should be possible? How would the command know what size it should display the timeline at in fullscreen?

I'm starting to think that this cannot be achieved properly with just the `size_min` options and that a third option is necessary to communicate that the timeline should be visible in a window and hidden in fullscreen by default.

I can also make an issue out of this.

**Reply by FichteFoll** [reply]
Converted to an issue: https://github.com/tomasklaen/uosc/issues/630

**Comment by FichteFoll** [comment, marked as answer]
With 5.0.0, the following configuration works as expected (via #571 and #661):

```
[protocol.http]
script-opts-append=uosc-progress=never
```

Thanks!
